feature-detects/scroll-to-options.js

```diff
@@ -0,0 +1,27 @@
+/*!
+{
+  "name": "scrollToOptions dictionary",
+  "property": "scrolltooptions"
+}
+!*/
+define(['Modernizr'], function (Modernizr) {
```

Comment:
Please use helper classes like this:

define(['Modernizr', 'createElement', 'getBody'], function (Modernizr, createElement, getBody) {